soh/soh/mixer.c

```diff
@@ -1,9 +1,22 @@
 #include <stdbool.h>
 #include <stdint.h>
 #include <string.h>
+#include <stdio.h>
 
 #include "mixer.h"
 
+#if defined(__SSE2__) || defined(__aarch64__)
+  #define SSE2_AVAILABLE
+#else
+  #pragma message("Warning: SSE2 support is not available. Code will not compile")
+#endif
+
+#if defined(__SSE2__)
+  #include <emmintrin.h>
+#elif defined(__aarch64__)
+  #include "sse2neon.h"
+#endif
```

Comment:
Looking for ideas: what's the best way to handle this considering we ship binaries?

Reply:
I think it's OK to assume any x86 target will support SSE2 since this feature came out in 2000 for Intel and 2003 for AMD. If that's no good they could look at how I handled SSE4.2's CRC32 instruction in FastCRC.c